crates/router/src/core/payments/flows/complete_authorize_flow.rs

```diff
@@ -112,10 +113,14 @@ impl Feature<api::CompleteAuthorize, types::CompleteAuthorizeData>
                 &payments::TokenizationAction::TokenizeInConnector,
                 self,
                 types::PaymentMethodTokenizationData::try_from(request)?,
+                should_continue_payment,
             )
             .await
         } else {
-            Ok(None)
+            Ok(types::PaymentMethodTokenResult {
+                payment_method_token_result: Ok(None),
+                is_payment_method_tokenization_performed: true,
+            })
```

Comment:
should this be true? as we are not performing any tokenization in case the connector is not payme